controllers/comments-controllers.js

```diff
@@ -1,9 +1,16 @@
+const { selectArticleById } = require("../models/articles-models");
 const { selectCommentsByArticleId } = require("../models/comments-models");
 
 exports.getCommentsByArticleId = (req, res, next) => {
-    const {article_id} = req.params;
-    selectCommentsByArticleId(article_id).then((comments)=>{
-        res.status(200).send({comments});
+    const { article_id } = req.params;
+    const promises = [selectCommentsByArticleId(article_id), selectArticleById(article_id)];
+
+    Promise.all(promises).then((promisesReturned)=>{
+        const comments = promisesReturned[0];
+        if(comments.length === 0){
+            res.sendStatus(204);    
```

Comment:
We are sending back an empty array if there are no comments - this still counts as content and a 200 should be sent back.

Reply:
I will fix that now, thank you